tap_github/client.py

```diff
@@ -231,7 +241,29 @@ def extract_repos_from_config(self):
         """
         repo_paths = list(filter(None, self.config['repository'].split(' ')))
 
-        orgs_with_all_repos = list(filter(lambda x: x.split('/')[1] == '*', repo_paths))
+        visited = set()
+        # Insert the duplicate repos found in the config repo_paths into duplicates
+        duplicates = [x for x in repo_paths if x in visited or (visited.add(x) or False)]
+        if duplicates:
+            LOGGER.warn('Duplicate repositories found: {} and will be synced only once.'.format(duplicates))
+
+        repo_paths = list(set(repo_paths))
+
+        orgs_with_all_repos = []
+        repos_with_errors = []
+        for each_repo in repo_paths:
+            # Split the repo_path by `/`.
+            split_repo_path = each_repo.split('/')
+            if len(split_repo_path) > 1 and split_repo_path[1] != '' and split_repo_path[0] != '':
+                if split_repo_path[1] == '*':
+                    orgs_with_all_repos.append(each_repo)
+            else:
+                # If `/` not found or repo name not found, append the repo_path in the repos_with_errors
```

Comment:
Also, add like `Organization not found` too.

Reply:
Updated the comment